api/swagger.yml

```diff
@@ -2383,6 +2389,14 @@ paths:
           required: false
           schema:
             type: string
+        - in: header
+          name: If-None-Match
+          description: Currently supports only "*" to allow uploading an object only if one doesn't exist yet
+          example: "*"
+          required: false
+          schema:
+            type: string
+            pattern: '^\*$'  # Currently, only "*" is supported
```

Comment:
@nopcoder I hope our generated clients don't validate.  Otherwise when we add other values, an old client library won't work with an application that wants to use a non-`*` value.